frontend/src/browser-router.js

```diff
@@ -1,12 +1,11 @@
-import { createBrowserRouter } from "react-router-dom";
-
-import TextField from '@mui/material/TextField';
+import { createBrowserRouter } from 'react-router-dom';
+import SendForm from './SendForm.js';
 
 const browserRouter = createBrowserRouter([
     {
-        path: "/documents/new",
+        path: '/documents/new',
         element: <div>
-            <TextField id="outlined-basic" label="Outlined" variant="outlined" />
+            <SendForm />
         </div>,
```

Comment:
Remove divs please, Let the page manage it's own structure. Also nesting your apps html is NOT well known but bad practice(At least according to google's lighthouse which is a tool for estimating the performance of your web app)